contracts/UsingWitnetBytes.sol

```diff
@@ -38,9 +38,9 @@ contract UsingWitnetBytes {
   * @param _id The id of a request that has been previously sent to the WitnetRequestsBoard.
   * @return A boolean telling if the request has been already accepted or not. `false` do not mean rejection, though.
   */
-  function witnetCheckRequestAccepted(uint256 _id) internal view returns(bool) {
+  function witnetCheckRequestAccepted(uint256 _id) internal returns(bool) {
```

Comment:
It is possible to keep the `view` modifier?

If I am not mistaken, this function uses `readDrHash` and then `getController`. 
I think, if we set `getController` to view it seems we could keep the `view` modifier.